kubernetes/deployment.template.yaml

```diff
@@ -0,0 +1,197 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ${SERVICE_NAME}
+  namespace: ${NAMESPACE}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ${SERVICE_NAME}
+  template:
+    metadata:
+      labels:
+        app: ${SERVICE_NAME}
+    spec:
+      initContainers:
+        - name: init-wait-db
+          image: busybox
+          command: ['sh', '-c', 'for i in $(seq 1 300); do nc -zvw1 shared-database 5432 && exit 0 || sleep 3; done; exit 1']
+      containers:
+        - image: ${IMAGE_NAME}
+          name: ${SERVICE_NAME}
+          env: #TODO Are these env variables correct? Where did he get these values from, application.yaml?
```

Comment:
See:
https://github.com/hmcts/rpa-em-annotation-api/pull/5/files#diff-94d15c95f467ca128570c4629136bc51R149

I assume it came from cmc-claim-store

looks correct

Reply:
@timja I'm going to recreate the changes in this PR for a different repo (native pdf annotator api), how will I know which env variables to import? Is it just the same ones, but with any "emannotationapp" and "shared-database" values switched to the correct name for that repo?

Reply:
whatever is in your main.tf app settings basicly